pmesh/_domain.pyx

```diff
@@ -92,7 +92,7 @@ def gridnd_fill(
                         kk = kk - 1
                     else:
                         break
-                k = k + 1
+            k = k + 1
```

Comment:
No. k shall increase only after we have put a new target into the list. If k is increased everytime the target_list will contain undefined numbers. There is another bug that i fixed in the last commit -- 1642d675a5e5a57ea5f2017bfdbba972d8adaa7d I think that's the one you are looking for. The cases wasn't covered by test cases. I added tests and tagged a new release.

Reply:
oh okay! I'll test that out now

Reply:
yep that fixed it...I'll close this